salt/tests/unit/modules/test_metalk8s_network.py

```diff
@@ -0,0 +1,87 @@
+from salt.exceptions import CommandExecutionError
+
+from salttesting.mixins import LoaderModuleMockMixin
+from salttesting.unit import TestCase
+from salttesting.mock import MagicMock, patch
+
+import metalk8s_network
+
+
+class Metalk8sNetworkTestCase(TestCase, LoaderModuleMockMixin):
```

Comment:
So with this `TestCase` base class, we get `unittest`-style test cases? Not `pytest`-style?

Reply:
Yes

Reply:
OK, so everything related to fixtures is implemented through mixins, and assertions need to use the "right" methods.

I'm OK with this choice, especially if the tooling provided by Saltstack forces us to go this way. But I would like the squad @scality/metalk8s to acknowledge this decision before we move forward, as this is different from the other `pytest` suite we have.

Reply:
Looks pretty clear to me, at least with these simple tests.